core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -1521,6 +1534,19 @@ public boolean isAvailable(Iterable<RequestLogProperty> properties) {
             return true;
         }
 
+        @Nullable
+        @Override
+        public RequestLog getIfAvailable(RequestLogProperty... properties){
+            return isAvailable(properties) ? this : null;
```

Comment:
`isAvailable` is always true.

Reply:
Should I return the RequestLog class without using the isAvailable function?